packages/frontend/src/screens/app/screens/tournaments/screens/tournament/components/admin-check.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```

Comment:
Is it a good structure to have    "...screens/app/screens/tournaments/screens/tournament/components/admin-check.tsx"

By that I mean, we have a tournament in a screens in a tournament in a screens? I think the directory is kinda weird...

Reply:
Our structure is that for every screen there are "subscreens" that might be distinct in their purpose, therefore we structure our directories with screens being a subpart of its parent directorys functionality. I might agree though on that tournaments/screens/tournament might need a namechange to better reflect their function.